ckanext/harvester_dashboard/templates/harvester_dashboard/list.html

```diff
@@ -23,8 +23,8 @@ <h1>{{ _('Harvester Dashboard') }}</h1>
       {% set organization = harvest_source_info.organization %}
```

Comment:
Github won't let me leave a comment on the lines that aren't part of the diff, but the statements in lines 17-19 above that are using the `c` object are an example of where we should rather pass in variables with the `extra_vars` dict I mentioned above, and here access them directly (without the `c.`).